### PR TITLE
added S00

### PR DESCRIPTION
**Pull request by prakharcode**

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -1,4 +1,4 @@
-using AstroBase
```
Comment (giordano) [inline test/runtests.jl]: :confused:

Comment (prakharcode) [reply test/runtests.jl]: Drawbacks of coding 4:30 am in morning :laughing:

File: src/terms_s00.jl [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  -119.94e-6,
+-72574.09e-6,
+    27.70e-6,
+    15.61e-6
+)
+
+const s000 = (
+
+# 1-10
+(( 0,  0,  0,  0,  1,  0,  0,  0), -2640.73e-6,   0.39e-6 ),
```
Comment (helgee) [inline src/terms_s00.jl]: I would prefer if you split this into two separate tuples of tuples.

Comment (prakharcode) [reply src/terms_s00.jl]: (( 0,  0,  0,  0,  1,  0,  0,  0), (-2640.73e-6,   0.39e-6 )) like this?

Comment (helgee) [reply src/terms_s00.jl]:
```julia
const s000_1 = (
    ( 0,  0,  0,  0,  1,  0,  0,  0),
    ...
)

const s000_2 = (
    (-2640.73e-6,   0.39e-6),
    ...
)
```

or something like it.

Comment (helgee) [reply src/terms_s00.jl]: I am not sure whether it is a problem or not but your data structure would make iteration type-unstable.

```julia
a = s000[1]

typeof(a[1]) # NTuple{8,Int64}
typeof(a[2]) # Float64
typeof(a[3]) # Float64 

for x in a
    # x is type-unstable and performance will suffer
end
```

**Pull request merged**